src/windows/find_tools.rs

```diff
@@ -87,15 +87,12 @@ pub fn find_tool(target: &str, tool: &str) -> Option<Tool> {
     impl_::find_msvc_environment(tool, target)
         .or_else(|| impl_::find_msvc_15plus(tool, target))
         .or_else(|| impl_::find_msvc_14(tool, target))
-        .or_else(|| impl_::find_msvc_12(tool, target))
 }
 
 /// A version of Visual Studio
 #[derive(Debug, PartialEq, Eq, Copy, Clone)]
 #[non_exhaustive]
 pub enum VsVers {
-    /// Visual Studio 12 (2013)
-    Vs12,
```

Comment:
This is a public struct, so removing this would be a major breaking change. I don't want `cc` to need 2.0 for this.

Reply:
The variant could be `#[deprecated]` instead, especially since users already need a wildcard pattern for `#[non_exhaustive]` if they're matching this.

Reply:
Ah, sorry, I didn't realize that this was exported from the crate!
I've added back `Vs12` and marked it as deprecated.